lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -64,16 +62,13 @@
     private static final String OBJ_COPIER = "copier";
     private static final String OBJ_CREATOR = "creator";
     private static final String OBJ_NAME = "oname";
-    
-    private static final String SEARCH_OBJ_TYPE = "otype";
-    private static final String SEARCH_OBJ_TYPE_VER = "otypever";
 
     private HttpHost esHost;
     private String esUser;
     private String esPassword;
     private String indexNamePrefix;
+    private boolean mergeTypes = false;
```

Comment:
There's something very wrong here. I specifically removed this field in an earlier PR and your PR is putting it back - there must be something funky in your fork.